Modules/Core/Common/include/itkGaussianDerivativeOperator.hxx

```diff
@@ -41,15 +41,11 @@ GaussianDerivativeOperator<TPixel, VDimension, TAllocator>::GenerateCoefficients
 
 
   // Calculate scale-space normalization factor for derivatives
-  double norm;
+  double norm = 1.0;
   if (m_NormalizeAcrossScale)
   {
     norm = std::pow(m_Variance, m_Order / 2.0);
   }
-  else
-  {
-    norm = 1.0;
-  }
```

Comment:
Thanks! Honestly I would prefer:

    double norm = m_NormalizeAcrossScale ? std::pow(m_Variance, m_Order / 2.0) : 1.0;

Rather than first initializing by `-1.0` and then resetting to `std::pow(m_Variance, m_Order / 2.0)`.